test/unit/postal-code.js

```diff
@@ -57,4 +57,21 @@ describe('postalCode', function () {
       });
     });
   });
+
+  describe('custom min length', function () {
+    it('allows passing in a custom min length', function () {
+      expect(postalCode('123')).to.deep.equal({
```

Comment:
I'd be okay with explicitly testing it here and mentioning that in the expectation.